app/assets/stylesheets/components/gitter_floating.scss

```diff
@@ -1,21 +1,53 @@
-.chat-floating-btn {
-  background: $button-primary;
-  border-radius: 50%;
+.floating-dropdown {
   bottom: 20px;
-  box-shadow: 0 0 15px 2px rgba(0,0,0,0.2);
-  cursor: pointer;
-  padding: 1em;
   position: fixed;
-  right: 20px;
   z-index: 1000;
+  right: 20px;
+
+  .floating-dropdown-item {
```

Comment:
We've adopted the BEM methodology for organising our CSS into clean and reusable components. Its not on every page yet, but its getting there.

Can we change this to follow BEM please? This would be an element of the 
`.floating-dropdown` block so we would use a class like this in the view for it:
`floating-dropdown__item` 

Then in the style sheet we can change this line to `&__item`.